src/main/java/org/jas/service/config/LastfmConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.jas.service.config;
+
+public class LastFMConfig {
+    public static String getLastFMKey() {
```

Comment:
Instead of using `static` methods, let's create a `@Component` annotation, in this way Spring will handle the instance. Also the package should be `org.jas.config` meaning not part of `service`